src/content/patterns/creation-flows/creation-flows.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Creation flows
+---
+
+
+<ImageComponent cols="8">
+
+![Example](images/1.png)
+
+</ImageComponent>
+
+
+#### Status:
+
+Experimental
+
+## Options
+
+<AnchorLinks>
+
+- [Standard creation](#standard-creation)
+- [Multistep creation](#multistep-creation)
+- [confirmation](#confirmation)
+
+</AnchorLinks>
+
+
+## Standard Creation
+
+<AnchorLinks small>
+
+- [Modal](#modal)
+- [Side panel](#side-panel)
+- [Full screen](#full-screen)
+- [Advanced option](#advanced-option)
+
+</AnchorLinks>
+
+### Modal
+If there are only 1 or 2 options involved in the creation then use a small, standard modal, overlaying the content where the creation was initiated. Scrolling inside a small modal should be avoided. 
+
+<ImageComponent cols="8">
+
+![Example](images/2.png)
+
+</ImageComponent>
+
+
+### Side panel
+Consider using a side panel to take advantage of the added real-estate and keep the user in context of where they were working. You may choose to use either the slide in or slide over panels.
+
+
+<ImageComponent cols="8">
+
+![Example](images/3.png)
+
+</ImageComponent>
+
+### Full page
+Consider using this when wanting to take full advantage of the screen’s available real estate. 
```

Comment:
Maybe elaborate on the why you would want to take advantage of the full page. Like "best for when you have a longer creation process that involves multiple form items to be completed" or something a long those line. You hit on the why in the modal and side panel but not so much this one. The examples shows it but it would be nice to actually say it.